redash/query_runner/__init__.py

```diff
@@ -312,16 +312,14 @@ def add_limit_to_query(self, query):
         return str(parsed_query)
 
     def apply_auto_limit(self, query_text, should_apply_auto_limit):
+        queries = split_sql_statements(query_text)
         if should_apply_auto_limit:
-            queries = split_sql_statements(query_text)
             # we only check for last one in the list because it is the one that we show result
             last_query = queries[-1]
             if self.query_is_select_no_limit(last_query):
                 queries[-1] = self.add_limit_to_query(last_query)
-            return combine_sql_statements(queries)
-        else:
-            return query_text
-
+        return combine_sql_statements(queries)
+        
```

Comment:
its a flake error W293 blank line contains white-space
so removing the new line at the end of apply_auto_limit function will fix it

Reply:
If the `pre-commit` hook has been set up, then that should automatically format things correctly when the git commit is created:

https://github.com/getredash/redash/wiki/Local-development-setup#configuring-pre-commit

If that's not setup, you can manually run the `make format` command at any time to do the same thing.  You'll need the Python virtual environment to be activate (with everything installed) for it to work though.

Reply:
Thanks @justinclift. Should i copy the above changes to my branch, fix formatting and create pull request?
Asking this since the above changes by @alexStrickner00 are not present in any branch in getredash/redash repository.

Reply:
@manikBS I'd personally go with cherry-picking @alexStrickner00's changes across to your repo (eg in a branch or something) in order to keep the authorship correct, and add your stuff afterwards as new commits.  Then make a PR from that. :smile:

That should (in theory) make for a PR we can merge, and that has both @alexStrickner00 and yourself correctly credited as authors.

Reply:
submitted new PR with the above format fix https://github.com/getredash/redash/pull/6351